pkg/instrumentation/auto/annotation.go

```diff
@@ -23,9 +23,12 @@ const (
 	defaultAnnotationValue = "true"
 )
 
+// +kubebuilder:rbac:groups="",resources=namespaces,verbs=list;watch;update
```

Comment:
We will have to create a namespace mutating webhook and can move this in there once thats ready.